### PR TITLE
add config for angus 2024-02-15 deployment

### PR DESCRIPTION
**Pull request by melaniefeen**

### CODE REVIEW

File: config/angus-20240215T0000/deployment.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        "publisher_url": "https://secoora.org",
+        "references": "",
+        "sea_name": "North Atlantic Ocean",
+        "source": "Observational data from a profiling glider",
+        "summary": "SKIO Angus Glider deployment in the North Atlantic Ocean (January 2024)",
+        "support_email": "catherine.edwards@skio.uga.edu",
+        "support_name": "Tides Foundation",
+        "support_role": "fiscal",
+        "support_type": "private",
+        "title": "Angus Slocum Glider Dataset (January 2024)",
```
Comment (lukecampbell) [inline config/angus-20240215T0000/deployment.json:36]: Could you change January to February?

File: config/angus-20240215T0000/deployment.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        "platform_type": "Slocum Glider",
+        "processing_level": "Dataset taken from glider native file format and is provided as is with no expressed or implied assurance of quality assurance or quality control.",
+        "project": "Right Whale Monitoring",
+        "publisher_email": "dmac@secoora.org",
+        "publisher_name": "SECOORA Glider Observatory",
+        "publisher_url": "https://secoora.org",
+        "references": "",
+        "sea_name": "North Atlantic Ocean",
+        "source": "Observational data from a profiling glider",
+        "summary": "SKIO Angus Glider deployment in the North Atlantic Ocean (January 2024)",
```
Comment (lukecampbell) [inline config/angus-20240215T0000/deployment.json:31]: Could you change January to February here?

**Review by lukecampbell — CHANGES_REQUESTED** [review]

File: config/angus-20240215T0000/deployment.json (hunk 2 above)
Comment (melaniefeen) [reply config/angus-20240215T0000/deployment.json]: Yes!

File: config/angus-20240215T0000/deployment.json (hunk 1 above)
Comment (melaniefeen) [reply config/angus-20240215T0000/deployment.json]: Updated!

**Review by lukecampbell — APPROVED** [review]

**Pull request merged**